ui/src/app/job-list/job-list.component.ts

```diff
@@ -165,6 +166,7 @@ export class JobListComponent implements OnInit {
   }
 
   private onClientPaginate(e: PageEvent) {
+    this.settingsService.setPageSize(e.pageSize, this.projectId);
```

Comment:
This seems a bit chatty - perhaps compare to `this.pageSize` and only update if it has changed? This also means you won't be saving saving settings for someone who has never touched the page size controls.

Reply:
done